### PR TITLE
WIP add Benchmark and samples

### PR DESCRIPTION
**Pull request by MarkZither**
trying to use benchmarks to test for checking for load issues like "influxdb write backoff for 30 seconds"

added readme with some badges to show status and the bench marks.

would you consider enabling issues and contributing guidelines? I was caught out by master not being upto date with release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# serilog-sinks-influxdb
+A serilog sink that writes events to InfluxDB.
```
Comment (pada57) [inline README.md:2]: Maybe says that it use syslog format and referencing [influxdb docs](https://www.influxdata.com/blog/writing-logs-directly-to-influxdb/)

File: README.md [hunk 2]
```diff
@@ -3,9 +3,14 @@ A serilog sink that writes events to InfluxDB.
...
 
+| Branch   | Status         |
+| -------- | -------------- |
+|Master Branch|[![Build status](https://ci.appveyor.com/api/projects/status/fh0wk0ov2f86u1lw/branch/master?svg=true)](https://ci.appveyor.com/project/MarkZither/serilog-sinks-influxdb/branch/master)|
```
Comment (pada57) [inline README.md:11]: weird builds are failing as this error is only a warning for png file

**Review by pada57 — CHANGES_REQUESTED** [review]
As i forked project i was using branches for each feature and merging back to a "release" branch but not yet into master as i was doing PR for each feature branch.

Could you do a new PR creating a branch from Release for now ?

i think most of your adaptions with defaults and convertors are already there. I could use instead of CreateDatabase async method which i then use GetAwaiter().Wait() since i want it to be synchronous before sink object constructor is finished

File: README.md (hunk 2 above)
Comment (MarkZither) [reply README.md:11]: fixed in #4

File: README.md (hunk 1 above)
Comment (MarkZither) [reply README.md:2]: added in PR #8

**Pull request closed without merging**